x-pack/plugins/security_solution/public/app/home/index.tsx

```diff
@@ -44,9 +44,18 @@ interface HomePageProps {
 }
 
 const HomePageComponent: React.FC<HomePageProps> = ({ children }) => {
-  const { application } = useKibana().services;
+  const { application, overlays } = useKibana().services;
   const subPluginId = useRef<string>('');
   const { ref, height = 0 } = useThrottledResizeObserver(300);
+  const banners$ = overlays.banners.get$();
+  const [headerFixed, setHeaderFixed] = useState<boolean>(true);
+  const mainPaddingTop = useMemo(() => (headerFixed ? height : 0), [headerFixed, height]);
+
+  useEffect(() => {
+    const subscription = banners$.subscribe((banners) => setHeaderFixed(!banners.length));
```

Comment:
So the logic is "the header is fixed when there are no banners?"

Reply:
Yes that what we decide in the zoom metting

Reply:
A little bit more details here, only the banners coming from kibana.